act-rules-format.md

```diff
@@ -160,7 +160,7 @@ Refer to the [Rule Type](#rule-types) section for detailed definitions of the ru
 Accessibility Requirements Mapping {#accessibility-requirements-mapping}
 ------------------------------------------------------------------------
```

Comment:
I keep coming back to the idea that there are two types of secondary requirements. 

Type A: The "AAA contrast for the AA rule" example
1. On pass, even with no other issues, the SC may not be satisfied
2. On fail, the SC is not satisfied

Type B: The "Keyboard trap non-standard" example
1. On pass, assuming no other problems, the SC is satisfied
2. On fail, the SC

---

So what if we did this. A conformance requirement is something for which both the following conditions are true:
1. On pass, assuming no other problems, the SC is satisfied; and
2. On fail, the SC is not satisfied

If only one of those conditions is true it is a secondary requirement. If neither is true, then that is not a requirement for the rule. The tricky part I think is in that "assuming no other problems" wording. In the 1.0 version we hadn't found a good way to define that. I feel that this is the key here. If we can figure that one out, we can make these requirement types work.

Reply:
Re: "assuming no other problems", I removed "further testing is needed" for pass on a previous draft (July 20), but feedback was that it would be too significant a change from version 1.0.

Reply:
I definitely agree that separating into 3 types of mappings flows more naturally, but I'm not sure I see the difference between `assuming no other problems` and `further testing is needed`. I feel like they are basically stating the same thing: "There is some condition that may occur that this rule doesn't test". 

Could we call Type A `Failure Mappings` and Type B `Satisfy Mappings` after how they relate to a rule? Then our mappings would look like:

- Conformance Mapping: Pass -> Satisfy or further testing needed. Fail -> Not satisfied
- Failure Mapping: Pass -> Further testing needed, Fail -> Not satisfied
- Satisfy Mapping: Pass -> Satisfy or further testing needed, Fail -> Further testing needed

Reply:
I've added the last two bullets in Secondary (starts at line 222). 

But, for Trevor's Failure mapping, "AAA", pass -> not satisfied.
The Satisfy mapping is called "OR" for now.

Reply:
On the conceptual level, I like the 3 kinds of mappings here.
But I see a hard time coming to make the difference between "Conformance Mapping" and "Failure Mapping" because of the "assuming no other problems" that is underlying.

---
In a case like "Image accessible name is descriptive", it is fairly obvious to figure out that "if there are no other non-text content, then 1.1.1 is satisfied". It can still get a bit tricky to determine precisely what it means, but the intuition should be fairly obvious for everybody.

But, for example, we have a (long standing) open PR for checking that [links inside paragraphs have non-color indication](https://github.com/act-rules/act-rules.github.io/pull/1010). In that case the "no other problems (related to 1.4.1)" bit is much harder to get 🤔 I guess we can say that if color is not used anywhere else than for links 1.4.1 is satisfied, so the rule has a "Conformance Mapping" to 1.4.1. But I feel the amount of possible uses of color will make any attempt at really defining "assuming no other problem" quite painful.

Reply:
Another issue I see with that is for rules like "Image has a non-empty accessible name". The mapping to 1.1.1 is here only a Failure Mapping. So, it is similar to "color contrast (minimum)" having a Failure Mapping to 1.4.6; or "link in context" having a Failure Mapping to 2.4.9.

Since part of the goal of this change is too allow tools to **not report** the (AAA) Failure Mapping, I foresee an unexpected consequence that we'd accept tools who do not report 1.1.1 on "Image has non empty accessible name" failures (plus the rule would only have secondary mappings). Somehow, this feels very wrong.